### DISCUSSION TITLE
Not everyone wants to learn R! How do we get SAS programmers onboard with R?

### DISCUSSION THREAD

**Discussion by drorberel** [rinpharma/rinpharma-summit-2024#8] [roundtable]
### Proposal

Preserve knowledge transfer while transitioning from SAS programming into R. How to collaboratively include SAS programmers in the process, leveraging their clinical data domain expertise. 

### Expected impact

This roundtable will focus on the human side of getting SAS programmers to join the effort of transitioning into R. R is definitely not trivial for everyone. A multilingual team can better achieve the development of regulatory complaint code by complementing each other’s strong capabilities. How do we get SAS programmers to get onboard? Even if a SAS programmer is not interested in learning R as a tool, perhaps we can show them what they can do with R to tackle some of their pain points. Proprietary and open-source ‘Cultures’ may clash. How do we help both the organization, and individuals from each ‘culture’ to collaborate, and enjoy the fruits of both tools?

### Prior discussions/work

I am an independent consultant who helps organizations transition analytical tools from SAS to R. Over the years I successfully helped organizations of different sizes to master such transitions. While each organization has its own unique needs and structure, some common patterns are observed. Will be glad to share my experience with the community.

https://rinpharma.com/publication/rinpharma_366/
[David Keyes - How to Convince Your Teammates to Learn R](https://www.youtube.com/watch?v=3upS7K-ODhQ&list=PLzwtsyfF_Z4q8_noXAPh3JFSiT22gg9Gv&index=19)

### Would you be willing to potentially facilitate this discussion?

Yes definately!

**Comment by epijim** [comment]
I feel this is definitely a big topic and probably the most disruptive topic all our statistical programming groups are going through right now is this change management topic of learning R, but also other data science tools like git and dependency management. 

One comment I have seeing the content above is this pitches it as SAS vs R. I feel sometimes it's more 'SAS worked for 10+ years, I don't want to learn something new' e.g. it's more change in general over is one better than the other. I think it doesn't change the response much though - regardless we should get people excited about the new tool(s)

**Reply by katieigartua** [reply]
I have found it to be a challenge when collaborating and sharing code and best practices across companies that have different code platform usage patterns.

**Comment by pzhang-cims** [comment]
@drorberel Hi, thanks for raising this topic, from traditional way to open-source solution, we are flooded by the online materials. While lots of paper/materials are contributed for people who has already had R experience, we need a friendly guidance for people using R, especially in pharma. I attach my pharmaSUG paper this year and hope that can help serve as a reference. https://www.pharmasug.org/proceedings/2024/ST/PharmaSUG-2024-ST-381.pdf

**Comment by epijim** [comment]
This has been added to the agenda.
